examples/kitchen-sink/schema.sql

```diff
@@ -24,6 +24,9 @@ create table another_thing (
   thing_id         int references thing(id) on delete cascade
 );
 
+create view non_mutation_view as
+  select 1;
```

Comment:
I really want to merge this, but first could you please add a test for auto updatable views that can be mutated? That we can be sure this code never breaks in the future, or with different PostgreSQL versions :blush: